notebooks/py/osmFISH.py

```diff
@@ -3,33 +3,190 @@
+# EPY: ESCAPE %autoreload 2
+# EPY: END code
+
+# EPY: START code
+experiment = starfish.data.osmFISH(use_test_data=True)
+stack = experiment['fov_000']['primary']
+# EPY: END code
+
 # EPY: START markdown
-### Loading the data into Starfish
+### Load pysmFISH results
+# EPY: END markdown
+
+# EPY: START markdown
+#The Field of view that we've used for the test data corresponds to Aldoc, imaged in round one, in position 33. We've also packaged the results from the osmFISH publication for this target to demonstrate that starfish is capable of recovering the same results. 
+# EPY: END markdown
+
+# EPY: START markdown
+#The below commands parse and load the results from this file. 
 # EPY: END markdown
 
 # EPY: START code
-# EPY: ESCAPE %gui qt5
+def load_results(pickle_file):
+    with open(pickle_file, 'rb') as f:
+        return pickle.load(f)
+
+def get_benchmark_peaks(loaded_results, redo_flag=False):
+
+    if not redo_flag:
+        sp = pd.DataFrame(
+            {
+                'y':loaded_results['selected_peaks'][:,0],
+                'x':loaded_results['selected_peaks'][:,1],
+                'selected_peaks_int': loaded_results['selected_peaks_int'],
+            }
+        )
+    else:
+        p = peaks(loaded_results)
+        coords = p[p.thr_array==loaded_results['selected_thr']].peaks_coords
+        coords = coords.values[0]
+        sp = pd.DataFrame({'x':coords[:,0], 'y':coords[:,1]})
+
+    return sp
+
+benchmark_results = load_results('./data/EXP-17-BP3597_hyb1_Aldoc_pos_33.pkl')
+benchmark_peaks = get_benchmark_peaks(benchmark_results, redo_flag=False)
+# EPY: END code
+
+# EPY: START markdown
+## Re-produce pysmFISH Results
+# EPY: END markdown
+
+# EPY: START markdown
+### Filtering code
+# EPY: END markdown
+
+# EPY: START markdown
+#Remove background using a gaussian high-pass filter, then enhance spots with a Laplacian filter.
+# EPY: END markdown
+
+# EPY: START code
+filter_ghp = starfish.image.Filter.GaussianHighPass(sigma=(1,8,8), is_volume=True)
+filter_laplace = starfish.image.Filter.Laplace(sigma=(0.2, 0.5, 0.5), is_volume=True)
+
+stack_ghp = filter_ghp.run(stack, in_place=False)
+stack_ghp_laplace = filter_laplace.run(stack_ghp, in_place=False)
 # EPY: END code
 
+# EPY: START markdown
+#Max project over Z, then select the 1st `(0)` channel for visualization in the notebook to demonstrate the effect of background removal using these filters. 
+# EPY: END markdown
+
 # EPY: START code
-from starfish import data, FieldOfView
-from starfish.types import Indices
-import os
+mp = stack_ghp_laplace.max_proj(Indices.Z)
+array_for_visualization = mp.xarray.sel({Indices.CH: 0}).squeeze()
 # EPY: END code
 
+# EPY: START code
+plt.figure(figsize=(10, 10))
+plt.imshow(
+    array_for_visualization, 
+    cmap='gray', 
+    vmin=np.percentile(array_for_visualization, 98), 
+    vmax=np.percentile(array_for_visualization, 99.9),
+)
+plt.title('Filtered max projection')
+plt.axis('off');
```

Comment:
```suggestion
plt.axis('off')
```

Reply:
Without a last-line `;`, irrelevant information about the axes is printed to the output by matplotlib.

Reply:
If you would prefer, the alternative is `_ = plt.axis("off")`.